samples/csharp/getting-started/MulticlassClassification_AutoML/MNIST/Program.cs

```diff
@@ -65,9 +65,10 @@ public static void Train(MLContext mlContext)
                 // STEP 3: Run an AutoML multiclass classification experiment
                 ConsoleHelper.ConsoleWriteHeader("=============== Running AutoML experiment ===============");
                 Console.WriteLine($"Running AutoML multiclass classification experiment for {ExperimentTime} seconds...");
+                var preFeaturizer = mlContext.Transforms.Conversion.MapValueToKey("Number", "Number", keyOrdinality: Microsoft.ML.Transforms.ValueToKeyMappingEstimator.KeyOrdinality.ByValue);
```

Comment:
I don't think you need the sort order specified. But you'll want a `MapKeyToValue` at the end. These should be automatically generated for you by the AutoML.

Reply:
The default is by occurrence if the parameter is unspecified. AutoML does automatically generate the key mappings however it too is by occurrence. Confirmed by opening the Model.zip generated by the master branch and checking `Vocabulary/Terms.txt` of the transform
```
# Number of terms of type 'Single' = 10
0	0
1	7
2	4
3	6
4	2
5	5
6	8
7	1
8	9
9	3
```
Which looking in `optdigits-train.csv` corresponds to order of appearance. Adding the prefeaturizer it seems AutoMl still generates the `MapKeyToValue` at the end (confirmed by `LastEstimator` field on the best model), though it's an assumption on my part that AutoML will not "rekey" a key column and that `MapKeyToValue`  corresponds to the prefeaturizer transform.

Reply:
@justinormont - Is this PR about AutoML API tested and ready to be merged?

Reply:
@daholste: Do we automatically append a `MapKeyToValue()` at the end of the AutoML pipeline to produce the original Label values (even with numeric inputs)? If not, we likely should be.

Reply:
yeah, we do append MapKeyToValue() at the end of the AutoML pipeline

Reply:
Since we add the `MapKeyToValue()` to the pipeline, the original values should be printed. 

Adding the `KeyOrdinality.ByValue` is an interesting method to ensure the internal representation also matches the file. Though this method only works specifically for this dataset as the classes are numeric and start at 0 and are not missing any values. 

If the class values aren't matching after the `MapKeyToValue()`, we should investigate that.
Basically, the `MapKeyToValue()` is the right way to obtain the original class values back. The purposed method will not work for the general case, so I wouldn't put it in a sample.